api/src/main/java/org/datacleaner/api/RestrictedFunctionalityCallToAction.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.api;
+
+import java.io.Serializable;
+
+/**
+ * Represents a "Call to Action" for users when restricted functionality is
+ * blocked, typically with the use of {@link RestrictedFunctionalityException}
+ * or {@link RestrictedFunctionalityMessage}.
+ */
+public class RestrictedFunctionalityCallToAction implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private final String _name;
```

Comment:
I was thinking something along the same line when I created it also ... Hmm maybe a solution could be to have a subclass for our most typical restrictions. That subclass could have the standard texts in there as well. Because I do see one good thing about the open-ended string which is that it will allow other vendors to make use of this construction too. This could be our sister companies within Neopost or even third parties.